minid_client/__init__.py

```diff
@@ -0,0 +1 @@
+__VERSION__ = '1.1.0'
```

Comment:
I added the extra 'patch' version number to make it semantic, but this isn't a requirement for PyPi and I would be happy to remove it.